roles/ansible-os-hardening/tasks/sysctl.yml

```diff
@@ -1,15 +1,15 @@
 ---
-- name: Only enable IP traffic forwarding, if required.
+- name: Disable IPv4 traffic forwarding.
   sysctl: name='net.ipv4.ip_forward' value=0 sysctl_set=yes state=present reload=yes ignoreerrors=yes
```

Comment:
‎Yes, you're right. Though i think we should do this in separate PR.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        Von: Christoph HartmannGesendet: Montag, 1. Juni 2015 14:00An: hardening-io/ansible-os-hardeningAntwort an: hardening-io/ansible-os-hardeningCc: Sebastian GumprichBetreff: Re: [ansible-os-hardening] Make tasks clearer (#13)In roles/ansible-os-hardening/tasks/sysctl.yml:
> @@ -1,15 +1,15 @@
>  ---
> -- name: Only enable IP traffic forwarding, if required.
> +- name: Disable IPv4 traffic forwarding.
>    sysctl: name='net.ipv4.ip_forward' value=0 sysctl_set=yes state=present reload=yes ignoreerrors=yes

we should add a  when: os_network_forwarding == false here

—Reply to this email directly or view it on GitHub.